scripts/policy/protocols/conntuple/vlan.zeek

```diff
@@ -0,0 +1,14 @@
+##! This script adapts Zeek's connection tuples to include 802.1Q VLAN and
+##! Q-in-Q tags, when available. Zeek normally ignores VLAN tags in its flow
+##! lookups; this change makes it factor them in and also makes those VLAN tags
+##! part of the conn_id record.
+
+redef record conn_id += {
+	## The outer VLAN for this connection, if applicable.
+	vlan: int      &log &optional;
```

Comment:
Is there any concern about name collisions if someone accidentally loads the script for multiple tuples?